cli/src/main/java/hudson/cli/CLIConnectionFactory.java

```diff
@@ -36,4 +36,19 @@ public CLIConnectionFactory basicAuth(String userInfo) {
         return authorization("Basic " + Base64.getEncoder().encodeToString(userInfo.getBytes()));
     }
 
+    /**
+     * Convenience method to call {@link #authorization} with the HTTP bearer authentication.
+     * Cf. {@code BasicHeaderApiTokenAuthenticator}.
+     */
+    public CLIConnectionFactory bearerAuth(String bearerToken) {
+        return authorization("Bearer " + bearerToken);
```

Comment:
Is it possible that the bearer token here is empty ?

Reply:
From my understanding of CLI class, it can be an empty string but not null -- follows the semantics of basicAuth